### DISCUSSION TITLE
Dev dependencies

### DISCUSSION THREAD

**Discussion by giscus** [isabelroses/website#28] [General]
# Dev dependencies

Nix is truly an enlightening experience

https://isabelroses.com/blog/nix-shells-8

**Comment by Minion3665** [comment]
> What if you also require all the dependencies of another program? I've personally never needed this, but you can do it like so

Thanks! I didn't know about `inputsFrom` but am pretty sure it'll make something I want to write ... if not easier then at least nicer to read
